docs/user/metacat/source/authinterface.rst

```diff
@@ -1,112 +1,19 @@
-
-  cd $METACAT/WEB-INF/scripts/bash/ 
-  
-In order to run the file, you must make the file executable::
-
-  chmod u+x authFileManager.sh 
-
-You run the command as the owner of the file::
-
-  ./authFileManager.sh  [options]
-
-Usage of the utility:
-
-./authFileManager.sh useradd -i -dn <user-distinguish-name> [-g <group-name> -e <email-address> -s <surname> -f <given-name> -o <organizationName>] 
-
-./authFileManager.sh useradd -h <hashed-password> -dn <user-distinguish-name> [-g <group-name> -e <email-address> -s <surname> -f <given-name> -o <organizationName>]
-
-./authFileManager.sh groupadd -g <group-name> [-d <description>] 
-
-./authFileManager.sh usermod -password -dn <user-distinguish-name> -i 
-
-./authFileManager.sh usermod -password -dn <user-distinguish-name> -h <new-hashed-password> 
-
-./authFileManager.sh usermod -group -a -dn <user-distinguish-name> -g <added-group-name> 
-
-./authFileManager.sh usermod -group -r -dn <user-distinguish-name> -g <removed-group-name> 
-
-
-.. Note:: 
-  
-  Metacat currently uses Bcrypt algorithm to hash the password. The hashed password following the "-h" should be generated by a Bcrypt algorithm. 
-  The hash string usually contains $ signs which can interfere with the command line arguments. You should use two SINGLE quotes to wrap the entire hashed string.
-  
-  The <user-distinguish-name> must look like "uid=john,o=something,dc=something,dc=something" and the group-name must look like "cn=dev,o=something,dc=something,dc=something".
-  
-  If an option value has spaces, the value should be enclosed in double quotes.
-  For example: ./authFileManager.sh groupadd -g cn=dev,o=something,dc=something,dc=something -d "Developers at NCEAS"
-  
-  The "-d <description>" option in the "groupadd" command is optional; 
-  "-g <groupname> -e <email-address> -s <surname> -f <given-name> -o <organizationName>" in the "useradd" command are optional as well.
-
-LDAP-Based Authentication
-----------------------------------
-Before the Metacat 2.4.0 release, LDAP was the default authentication mechanism and was configured to use 
-the NCEAS LDAP server. We are now restricting access to the server to only trusted partners who can 
-guarantee secure communication with their clients and the LDAP server. 
-If you are not on the list, you can contact us for more information or you may use the password file authentication
-(for a small group of users) or set up your own LDAP server (for a big group of users).
+If ORCID authentication is not suitable for your deployment, a custom authentication mechanism
+can be built. Metacat is written such that this Authentication provider is replaceable with
+another class that implements the same interface (``AuthInterface``). As an Administrator, you have
+the choice to provide an alternative implementation of ``AuthInterface`` and then configuring
+**metacat-site.properties** to use that class for authentication instead of ORCID authentication.
```

Comment:
I'm not sure this level of detail is appropriate for metacat admins - they will typically be scientists, not developers. We could maybe put this in chapter 18?

Reply:
I agree - it's likely going to add more confusion. This will be moved to the developers section.